packages/plugins/integration-plane-ui/src/lib/components/plane-settings/plane-settings.component.html

```
@@ -0,0 +1,134 @@
+<nb-card class="card-scroll">
+	<nb-card-header class="d-flex align-items-center justify-content-between">
+		<div class="d-flex align-items-center">
+			<ngx-back-navigation></ngx-back-navigation>
+			<h5 class="mb-0">{{ 'INTEGRATIONS.PLANE_PAGE.TITLE' | translate }}</h5>
+		</div>
+		<div class="header-actions">
+			@if (!isEditing()) {
+				<button
+					nbButton
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)